lib/saml/url_service.rb

```diff
@@ -168,7 +176,8 @@ def build_sso_url(link_authn_context)
 
     def build_authn_context(assurance_level_url, identity_provider = Settings.saml_ssoe.idme_authn_context)
       if identity_provider
-        [assurance_level_url, identity_provider]
+        assurance_level_url = [assurance_level_url] unless assurance_level_url.is_a?(Array)
```

Comment:
For now, I think maybe we shouldn't touch `build_authn_context`, and just add a separate function, `build_logingov_authn_context`, that we can refactor later. Since we're looking to merge these changes without being able to properly test, I'd prefer if we lessened the chance to negatively impact existing login methods